### ISSUE TITLE
Error during conversion to Float

### ISSUE THREAD

**Issue by puddingfactory** [puddingfactory/usaa-ynab#1]
Final column of record seems to contain `\r`, causing parser to error.

```
Error encountered while converting a value from alpha to float: strconv.ParseFloat: parsing "-82.65\r": invalid syntax
```

**Comment by puddingfactory** [comment]
As I was writing this, I realized that I don't need to convert from string to float to begin with (was using it to determine if num < 0 so I could flip it to a positive).

I just need to check and see if the first character is `-` or not... if it is, remove that character... am disappointed it took me so long to see that this was a better solution.

**Comment by puddingfactory** [comment]
Fixed in v0.3b
